+ source /root/mugen-riscv/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen-riscv/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start to prepare the database config.'
+ message='Start to prepare the database config.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to prepare the database config.'
Sat Nov 12 01:57:35 2022 - INFO  - Start to prepare the database config.
+ DNF_INSTALL clamav
+ pkgs=clamav
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen-riscv/libs/locallibs/rpm_manage.py install --pkgs clamav --node 1 --tempfile ''
Sat Nov 12 01:57:48 2022 - INFO  - pkgs:(clamav) is already installed
+ tmpfile2=
+ '[' -z '' ']'
+ tmpfile=
+ mkdir test_virus_collection
mkdir: cannot create directory ‘test_virus_collection’: File exists
+ echo test1
+ echo test2
+ cp /var/lib/clamav/main.cvd test_virus_collection/
+ LOG_INFO 'End to prepare the database config.'
+ message='End to prepare the database config.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End to prepare the database config.'
Sat Nov 12 01:57:52 2022 - INFO  - End to prepare the database config.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Sat Nov 12 01:57:53 2022 - INFO  - Start to run test.
+ clamscan --max-recursion 1024 --max-dir-recursion 10 --max-embeddedpe 10 --max-htmlnormalize 10
LibClamAV Warning: **************************************************
LibClamAV Warning: ***  The virus database is older than 7 days!  ***
LibClamAV Warning: ***   Please update it as soon as possible.    ***
LibClamAV Warning: **************************************************
+ CHECK_RESULT 0 0 0 'Check clamscan --max-recursion 1024 --max-dir-recursion 10 --max-embeddedpe 10 --max-htmlnormalize 10 failed.'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Check clamscan --max-recursion 1024 --max-dir-recursion 10 --max-embeddedpe 10 --max-htmlnormalize 10 failed.'
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ clamscan --allmatch test_virus_collection --include=REGEX --include-dir=REGEX --max-htmlnotags 5 --max-scriptnormalize 5
oe_test_clamav_clamscan_3.sh: line 38: /usr/bin/clamscan: No such file or directory
+ CHECK_RESULT 127 0 0 'Check clamscan --allmatch test_virus_collection --include=REGEX --include-dir=REGEX --max-htmlnotags 5 --max-scriptnormalize 5 failed.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check clamscan --allmatch test_virus_collection --include=REGEX --include-dir=REGEX --max-htmlnotags 5 --max-scriptnormalize 5 failed.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check clamscan --allmatch test_virus_collection --include=REGEX --include-dir=REGEX --max-htmlnotags 5 --max-scriptnormalize 5 failed.'
+ LOG_ERROR 'Check clamscan --allmatch test_virus_collection --include=REGEX --include-dir=REGEX --max-htmlnotags 5 --max-scriptnormalize 5 failed.'
+ message='Check clamscan --allmatch test_virus_collection --include=REGEX --include-dir=REGEX --max-htmlnotags 5 --max-scriptnormalize 5 failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check clamscan --allmatch test_virus_collection --include=REGEX --include-dir=REGEX --max-htmlnotags 5 --max-scriptnormalize 5 failed.'
Sat Nov 12 01:59:27 2022 - ERROR - Check clamscan --allmatch test_virus_collection --include=REGEX --include-dir=REGEX --max-htmlnotags 5 --max-scriptnormalize 5 failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_clamav_clamscan_3.sh line 39'
+ message='oe_test_clamav_clamscan_3.sh line 39'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_clamav_clamscan_3.sh line 39'
Sat Nov 12 01:59:28 2022 - ERROR - oe_test_clamav_clamscan_3.sh line 39
+ return 0
+ clamscan -z test_virus_collection --cross-fs /opt --max-filesize 1024 --max-ziptypercg 5
oe_test_clamav_clamscan_3.sh: line 40: /usr/bin/clamscan: No such file or directory
+ CHECK_RESULT 127 0 0 'Check clamscan -z test_virus_collection --cross-fs /opt --max-filesize 1024 --max-ziptypercg 5 failed.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check clamscan -z test_virus_collection --cross-fs /opt --max-filesize 1024 --max-ziptypercg 5 failed.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check clamscan -z test_virus_collection --cross-fs /opt --max-filesize 1024 --max-ziptypercg 5 failed.'
+ LOG_ERROR 'Check clamscan -z test_virus_collection --cross-fs /opt --max-filesize 1024 --max-ziptypercg 5 failed.'
+ message='Check clamscan -z test_virus_collection --cross-fs /opt --max-filesize 1024 --max-ziptypercg 5 failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check clamscan -z test_virus_collection --cross-fs /opt --max-filesize 1024 --max-ziptypercg 5 failed.'
Sat Nov 12 01:59:29 2022 - ERROR - Check clamscan -z test_virus_collection --cross-fs /opt --max-filesize 1024 --max-ziptypercg 5 failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_clamav_clamscan_3.sh line 41'
+ message='oe_test_clamav_clamscan_3.sh line 41'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_clamav_clamscan_3.sh line 41'
Sat Nov 12 01:59:30 2022 - ERROR - oe_test_clamav_clamscan_3.sh line 41
+ return 0
+ clamscan --follow-dir-symlinks 0 --max-scansize 1024
oe_test_clamav_clamscan_3.sh: line 42: /usr/bin/clamscan: No such file or directory
+ CHECK_RESULT 127 0 0 'Check clamscan --follow-dir-symlinks 0 --max-scansize 1024 failed.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check clamscan --follow-dir-symlinks 0 --max-scansize 1024 failed.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check clamscan --follow-dir-symlinks 0 --max-scansize 1024 failed.'
+ LOG_ERROR 'Check clamscan --follow-dir-symlinks 0 --max-scansize 1024 failed.'
+ message='Check clamscan --follow-dir-symlinks 0 --max-scansize 1024 failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check clamscan --follow-dir-symlinks 0 --max-scansize 1024 failed.'
Sat Nov 12 01:59:31 2022 - ERROR - Check clamscan --follow-dir-symlinks 0 --max-scansize 1024 failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_clamav_clamscan_3.sh line 43'
+ message='oe_test_clamav_clamscan_3.sh line 43'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_clamav_clamscan_3.sh line 43'
Sat Nov 12 01:59:32 2022 - ERROR - oe_test_clamav_clamscan_3.sh line 43
+ return 0
+ clamscan --follow-file-symlinks 0 --max-files 1024 --max-ziptypercg 5
oe_test_clamav_clamscan_3.sh: line 44: /usr/bin/clamscan: No such file or directory
+ CHECK_RESULT 127 0 0 'Check clamscan --follow-file-symlinks 0 --max-files 1024 --max-ziptypercg 5 failed.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check clamscan --follow-file-symlinks 0 --max-files 1024 --max-ziptypercg 5 failed.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check clamscan --follow-file-symlinks 0 --max-files 1024 --max-ziptypercg 5 failed.'
+ LOG_ERROR 'Check clamscan --follow-file-symlinks 0 --max-files 1024 --max-ziptypercg 5 failed.'
+ message='Check clamscan --follow-file-symlinks 0 --max-files 1024 --max-ziptypercg 5 failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check clamscan --follow-file-symlinks 0 --max-files 1024 --max-ziptypercg 5 failed.'
Sat Nov 12 01:59:34 2022 - ERROR - Check clamscan --follow-file-symlinks 0 --max-files 1024 --max-ziptypercg 5 failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_clamav_clamscan_3.sh line 45'
+ message='oe_test_clamav_clamscan_3.sh line 45'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_clamav_clamscan_3.sh line 45'
Sat Nov 12 01:59:35 2022 - ERROR - oe_test_clamav_clamscan_3.sh line 45
+ return 0
+ clamscan --file-list /opt --exclude-dir=REGEX --max-partitions 5 --max-iconspe 5
oe_test_clamav_clamscan_3.sh: line 46: /usr/bin/clamscan: No such file or directory
+ CHECK_RESULT 127 0 0 'Check clamscan --file-list /opt --exclude-dir=REGEX --max-partitions 5 --max-iconspe 5 failed.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check clamscan --file-list /opt --exclude-dir=REGEX --max-partitions 5 --max-iconspe 5 failed.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check clamscan --file-list /opt --exclude-dir=REGEX --max-partitions 5 --max-iconspe 5 failed.'
+ LOG_ERROR 'Check clamscan --file-list /opt --exclude-dir=REGEX --max-partitions 5 --max-iconspe 5 failed.'
+ message='Check clamscan --file-list /opt --exclude-dir=REGEX --max-partitions 5 --max-iconspe 5 failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check clamscan --file-list /opt --exclude-dir=REGEX --max-partitions 5 --max-iconspe 5 failed.'
Sat Nov 12 01:59:36 2022 - ERROR - Check clamscan --file-list /opt --exclude-dir=REGEX --max-partitions 5 --max-iconspe 5 failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_clamav_clamscan_3.sh line 47'
+ message='oe_test_clamav_clamscan_3.sh line 47'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_clamav_clamscan_3.sh line 47'
Sat Nov 12 01:59:37 2022 - ERROR - oe_test_clamav_clamscan_3.sh line 47
+ return 0
+ clamscan --bytecode test_virus_collection --bytecode-timeout 50 --max-rechwp3 5
oe_test_clamav_clamscan_3.sh: line 48: /usr/bin/clamscan: No such file or directory
+ CHECK_RESULT 127 0 0 'Check clamscan --bytecode test_virus_collection --bytecode-timeout 50 --max-rechwp3 5 failed.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check clamscan --bytecode test_virus_collection --bytecode-timeout 50 --max-rechwp3 5 failed.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check clamscan --bytecode test_virus_collection --bytecode-timeout 50 --max-rechwp3 5 failed.'
+ LOG_ERROR 'Check clamscan --bytecode test_virus_collection --bytecode-timeout 50 --max-rechwp3 5 failed.'
+ message='Check clamscan --bytecode test_virus_collection --bytecode-timeout 50 --max-rechwp3 5 failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check clamscan --bytecode test_virus_collection --bytecode-timeout 50 --max-rechwp3 5 failed.'
Sat Nov 12 01:59:38 2022 - ERROR - Check clamscan --bytecode test_virus_collection --bytecode-timeout 50 --max-rechwp3 5 failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_clamav_clamscan_3.sh line 49'
+ message='oe_test_clamav_clamscan_3.sh line 49'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_clamav_clamscan_3.sh line 49'
Sat Nov 12 01:59:39 2022 - ERROR - oe_test_clamav_clamscan_3.sh line 49
+ return 0
+ clamscan --bytecode-unsigned test_virus_collection --statistics bytecode --pcre-match-limit 5
oe_test_clamav_clamscan_3.sh: line 50: /usr/bin/clamscan: No such file or directory
+ CHECK_RESULT 127 0 0 'Check clamscan --bytecode-unsigned test_virus_collection --statistics bytecode --pcre-match-limit 5 failed.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check clamscan --bytecode-unsigned test_virus_collection --statistics bytecode --pcre-match-limit 5 failed.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check clamscan --bytecode-unsigned test_virus_collection --statistics bytecode --pcre-match-limit 5 failed.'
+ LOG_ERROR 'Check clamscan --bytecode-unsigned test_virus_collection --statistics bytecode --pcre-match-limit 5 failed.'
+ message='Check clamscan --bytecode-unsigned test_virus_collection --statistics bytecode --pcre-match-limit 5 failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check clamscan --bytecode-unsigned test_virus_collection --statistics bytecode --pcre-match-limit 5 failed.'
Sat Nov 12 01:59:41 2022 - ERROR - Check clamscan --bytecode-unsigned test_virus_collection --statistics bytecode --pcre-match-limit 5 failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_clamav_clamscan_3.sh line 51'
+ message='oe_test_clamav_clamscan_3.sh line 51'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_clamav_clamscan_3.sh line 51'
Sat Nov 12 01:59:42 2022 - ERROR - oe_test_clamav_clamscan_3.sh line 51
+ return 0
+ clamscan --detect-pua test_virus_collection --pcre-recmatch-limit 5 --phishing-sigs test_virus_collection --statistics pcre
oe_test_clamav_clamscan_3.sh: line 52: /usr/bin/clamscan: No such file or directory
+ CHECK_RESULT 127 0 0 'Check clamscan --detect-pua test_virus_collection  --pcre-recmatch-limit 5 --phishing-sigs test_virus_collection --statistics pcre failed.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check clamscan --detect-pua test_virus_collection  --pcre-recmatch-limit 5 --phishing-sigs test_virus_collection --statistics pcre failed.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check clamscan --detect-pua test_virus_collection  --pcre-recmatch-limit 5 --phishing-sigs test_virus_collection --statistics pcre failed.'
+ LOG_ERROR 'Check clamscan --detect-pua test_virus_collection  --pcre-recmatch-limit 5 --phishing-sigs test_virus_collection --statistics pcre failed.'
+ message='Check clamscan --detect-pua test_virus_collection  --pcre-recmatch-limit 5 --phishing-sigs test_virus_collection --statistics pcre failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check clamscan --detect-pua test_virus_collection  --pcre-recmatch-limit 5 --phishing-sigs test_virus_collection --statistics pcre failed.'
Sat Nov 12 01:59:43 2022 - ERROR - Check clamscan --detect-pua test_virus_collection  --pcre-recmatch-limit 5 --phishing-sigs test_virus_collection --statistics pcre failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_clamav_clamscan_3.sh line 53'
+ message='oe_test_clamav_clamscan_3.sh line 53'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_clamav_clamscan_3.sh line 53'
Sat Nov 12 01:59:44 2022 - ERROR - oe_test_clamav_clamscan_3.sh line 53
+ return 0
+ clamscan --include-pua=CAT --detect-structured test_virus_collection --structured-ssn-format=2 --pcre-max-filesize 5 --statistics none
oe_test_clamav_clamscan_3.sh: line 54: /usr/bin/clamscan: No such file or directory
+ CHECK_RESULT 127 0 0 'Check clamscan --include-pua=CAT --detect-structured test_virus_collection --structured-ssn-format=2 --pcre-max-filesize 5 --statistics none failed.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check clamscan --include-pua=CAT --detect-structured test_virus_collection --structured-ssn-format=2 --pcre-max-filesize 5 --statistics none failed.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check clamscan --include-pua=CAT --detect-structured test_virus_collection --structured-ssn-format=2 --pcre-max-filesize 5 --statistics none failed.'
+ LOG_ERROR 'Check clamscan --include-pua=CAT --detect-structured test_virus_collection --structured-ssn-format=2 --pcre-max-filesize 5 --statistics none failed.'
+ message='Check clamscan --include-pua=CAT --detect-structured test_virus_collection --structured-ssn-format=2 --pcre-max-filesize 5 --statistics none failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check clamscan --include-pua=CAT --detect-structured test_virus_collection --structured-ssn-format=2 --pcre-max-filesize 5 --statistics none failed.'
Sat Nov 12 01:59:45 2022 - ERROR - Check clamscan --include-pua=CAT --detect-structured test_virus_collection --structured-ssn-format=2 --pcre-max-filesize 5 --statistics none failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_clamav_clamscan_3.sh line 55'
+ message='oe_test_clamav_clamscan_3.sh line 55'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_clamav_clamscan_3.sh line 55'
Sat Nov 12 01:59:46 2022 - ERROR - oe_test_clamav_clamscan_3.sh line 55
+ return 0
+ clamscan --structured-ssn-count 1 --structured-cc-count 1 --scan-mail test_virus_collection --disable-cache
oe_test_clamav_clamscan_3.sh: line 56: /usr/bin/clamscan: No such file or directory
+ CHECK_RESULT 127 0 0 'Check clamscan --structured-ssn-count 1 --structured-cc-count 1 --scan-mail test_virus_collection --disable-cache failed.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check clamscan --structured-ssn-count 1 --structured-cc-count 1 --scan-mail test_virus_collection --disable-cache failed.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check clamscan --structured-ssn-count 1 --structured-cc-count 1 --scan-mail test_virus_collection --disable-cache failed.'
+ LOG_ERROR 'Check clamscan --structured-ssn-count 1 --structured-cc-count 1 --scan-mail test_virus_collection --disable-cache failed.'
+ message='Check clamscan --structured-ssn-count 1 --structured-cc-count 1 --scan-mail test_virus_collection --disable-cache failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check clamscan --structured-ssn-count 1 --structured-cc-count 1 --scan-mail test_virus_collection --disable-cache failed.'
Sat Nov 12 01:59:47 2022 - ERROR - Check clamscan --structured-ssn-count 1 --structured-cc-count 1 --scan-mail test_virus_collection --disable-cache failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_clamav_clamscan_3.sh line 57'
+ message='oe_test_clamav_clamscan_3.sh line 57'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_clamav_clamscan_3.sh line 57'
Sat Nov 12 01:59:48 2022 - ERROR - oe_test_clamav_clamscan_3.sh line 57
+ return 0
+ clamscan --phishing-scan-urls test_virus_collection --heuristic-alerts test_virus_collection --heuristic-scan-precedence test_virus_collection
oe_test_clamav_clamscan_3.sh: line 58: /usr/bin/clamscan: No such file or directory
+ CHECK_RESULT 127 0 0 'Check clamscan --phishing-scan-urls test_virus_collection --heuristic-alerts test_virus_collection --heuristic-scan-precedence test_virus_collection failed.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check clamscan --phishing-scan-urls test_virus_collection --heuristic-alerts test_virus_collection --heuristic-scan-precedence test_virus_collection failed.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check clamscan --phishing-scan-urls test_virus_collection --heuristic-alerts test_virus_collection --heuristic-scan-precedence test_virus_collection failed.'
+ LOG_ERROR 'Check clamscan --phishing-scan-urls test_virus_collection --heuristic-alerts test_virus_collection --heuristic-scan-precedence test_virus_collection failed.'
+ message='Check clamscan --phishing-scan-urls test_virus_collection --heuristic-alerts test_virus_collection --heuristic-scan-precedence test_virus_collection failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check clamscan --phishing-scan-urls test_virus_collection --heuristic-alerts test_virus_collection --heuristic-scan-precedence test_virus_collection failed.'
Sat Nov 12 01:59:50 2022 - ERROR - Check clamscan --phishing-scan-urls test_virus_collection --heuristic-alerts test_virus_collection --heuristic-scan-precedence test_virus_collection failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_clamav_clamscan_3.sh line 59'
+ message='oe_test_clamav_clamscan_3.sh line 59'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_clamav_clamscan_3.sh line 59'
Sat Nov 12 01:59:51 2022 - ERROR - oe_test_clamav_clamscan_3.sh line 59
+ return 0
+ clamscan --normalize test_virus_collection --scan-pe test_virus_collection --alert-encrypted-doc test_virus_collection --alert-macros test_virus_collection
oe_test_clamav_clamscan_3.sh: line 60: /usr/bin/clamscan: No such file or directory
+ CHECK_RESULT 127 0 0 'Check clamscan --normalize test_virus_collection --scan-pe test_virus_collection --alert-encrypted-doc test_virus_collection --alert-macros test_virus_collection failed.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check clamscan --normalize test_virus_collection --scan-pe test_virus_collection --alert-encrypted-doc test_virus_collection --alert-macros test_virus_collection failed.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check clamscan --normalize test_virus_collection --scan-pe test_virus_collection --alert-encrypted-doc test_virus_collection --alert-macros test_virus_collection failed.'
+ LOG_ERROR 'Check clamscan --normalize test_virus_collection --scan-pe test_virus_collection --alert-encrypted-doc test_virus_collection --alert-macros test_virus_collection failed.'
+ message='Check clamscan --normalize test_virus_collection --scan-pe test_virus_collection --alert-encrypted-doc test_virus_collection --alert-macros test_virus_collection failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check clamscan --normalize test_virus_collection --scan-pe test_virus_collection --alert-encrypted-doc test_virus_collection --alert-macros test_virus_collection failed.'
Sat Nov 12 01:59:52 2022 - ERROR - Check clamscan --normalize test_virus_collection --scan-pe test_virus_collection --alert-encrypted-doc test_virus_collection --alert-macros test_virus_collection failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_clamav_clamscan_3.sh line 61'
+ message='oe_test_clamav_clamscan_3.sh line 61'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_clamav_clamscan_3.sh line 61'
Sat Nov 12 01:59:53 2022 - ERROR - oe_test_clamav_clamscan_3.sh line 61
+ return 0
+ clamscan --alert-phishing-ssl test_virus_collection --alert-phishing-cloak test_virus_collection --alert-partition-intersection test_virus_collection --nocerts --dumpcerts --max-scantime 100
oe_test_clamav_clamscan_3.sh: line 62: /usr/bin/clamscan: No such file or directory
+ CHECK_RESULT 127 0 0 'Check clamscan --alert-phishing-ssl test_virus_collection --alert-phishing-cloak test_virus_collection --alert-partition-intersection test_virus_collection --nocerts --dumpcerts --max-scantime 100 failed.'
+ actual_result=127
+ expect_result=0
+ mode=0
+ error_log='Check clamscan --alert-phishing-ssl test_virus_collection --alert-phishing-cloak test_virus_collection --alert-partition-intersection test_virus_collection --nocerts --dumpcerts --max-scantime 100 failed.'
+ '[' -z 127 ']'
+ '[' 0 -eq 0 ']'
+ test 127x '!=' 0x
+ test -n 'Check clamscan --alert-phishing-ssl test_virus_collection --alert-phishing-cloak test_virus_collection --alert-partition-intersection test_virus_collection --nocerts --dumpcerts --max-scantime 100 failed.'
+ LOG_ERROR 'Check clamscan --alert-phishing-ssl test_virus_collection --alert-phishing-cloak test_virus_collection --alert-partition-intersection test_virus_collection --nocerts --dumpcerts --max-scantime 100 failed.'
+ message='Check clamscan --alert-phishing-ssl test_virus_collection --alert-phishing-cloak test_virus_collection --alert-partition-intersection test_virus_collection --nocerts --dumpcerts --max-scantime 100 failed.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'Check clamscan --alert-phishing-ssl test_virus_collection --alert-phishing-cloak test_virus_collection --alert-partition-intersection test_virus_collection --nocerts --dumpcerts --max-scantime 100 failed.'
Sat Nov 12 01:59:54 2022 - ERROR - Check clamscan --alert-phishing-ssl test_virus_collection --alert-phishing-cloak test_virus_collection --alert-partition-intersection test_virus_collection --nocerts --dumpcerts --max-scantime 100 failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_clamav_clamscan_3.sh line 63'
+ message='oe_test_clamav_clamscan_3.sh line 63'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'oe_test_clamav_clamscan_3.sh line 63'
Sat Nov 12 01:59:55 2022 - ERROR - oe_test_clamav_clamscan_3.sh line 63
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Sat Nov 12 01:59:56 2022 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 13
+ test 13 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Sat Nov 12 01:59:57 2022 - ERROR - failed to execute the case.
+ exit 13
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Sat Nov 12 01:59:58 2022 - INFO  - Start to restore the test environment.
+ rm -rf test_clamscan.log test_virus_collection test_log.log
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z '' ]]
+ [[ -z '' ]]
+ LOG_WARN 'no thing to do.'
+ message='no thing to do.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level warn --message 'no thing to do.'
Sat Nov 12 01:59:59 2022 - WARN  - no thing to do.
+ return 0
+ LOG_INFO 'End to restore the test environment.'
+ message='End to restore the test environment.'
+ python3 /root/mugen-riscv/libs/locallibs/mugen_log.py --level info --message 'End to restore the test environment.'
Sat Nov 12 02:00:00 2022 - INFO  - End to restore the test environment.
